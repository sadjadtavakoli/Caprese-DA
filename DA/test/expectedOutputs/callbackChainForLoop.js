
#3 function arrowFunction-test/unit_tests/callbackchainforloop.js-3-19 entered from callbackChainForLoop.js-test/unit_tests/callbackchainforloop.js-1-19
#7 function arrowFunction-test/unit_tests/callbackchainforloop.js-7-17 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-3-19
#7 function arrowFunction-test/unit_tests/callbackchainforloop.js-7-17 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-3-19
#7 function arrowFunction-test/unit_tests/callbackchainforloop.js-7-17 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-3-19
#7 function arrowFunction-test/unit_tests/callbackchainforloop.js-7-17 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-3-19
#8 function arrowFunction-test/unit_tests/callbackchainforloop.js-8-16 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-7-17
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#8 function arrowFunction-test/unit_tests/callbackchainforloop.js-8-16 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-7-17
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#8 function arrowFunction-test/unit_tests/callbackchainforloop.js-8-16 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-7-17
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#8 function arrowFunction-test/unit_tests/callbackchainforloop.js-8-16 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-7-17
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#13 function arrowFunction-test/unit_tests/callbackchainforloop.js-13-14 entered from arrowFunction-test/unit_tests/callbackchainforloop.js-8-16
#end Execution