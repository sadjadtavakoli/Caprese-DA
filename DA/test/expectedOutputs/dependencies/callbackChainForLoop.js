{"arrowFunction-test/unit_tests/callbackchainforloop.js-3-19":{"callers":["t_1"],"tests":[]},"arrowFunction-test/unit_tests/callbackchainforloop.js-7-17":{"callers":["t_361"],"tests":[]},"arrowFunction-test/unit_tests/callbackchainforloop.js-8-16":{"callers":["t_55"],"tests":[]},"arrowFunction-test/unit_tests/callbackchainforloop.js-13-14":{"callers":["t_602"],"tests":[]},"keyMap":{"t_1":"callbackChainForLoop.js-test/unit_tests/callbackchainforloop.js-1-19","t_361":"arrowFunction-test/unit_tests/callbackchainforloop.js-3-19","t_55":"arrowFunction-test/unit_tests/callbackchainforloop.js-7-17","t_602":"arrowFunction-test/unit_tests/callbackchainforloop.js-8-16","t_63":"arrowFunction-test/unit_tests/callbackchainforloop.js-13-14"}}