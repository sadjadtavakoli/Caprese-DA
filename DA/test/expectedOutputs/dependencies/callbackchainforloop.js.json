{"keyMap":{"t_1":"callbackchainforloop.js-test/unit_tests/callbackchainforloop.js-1-19","t_31":"arrowAnonymousFunction-test/unit_tests/callbackchainforloop.js-3-19","t_70":"arrowAnonymousFunction-test/unit_tests/callbackchainforloop.js-7-17","t_102":"arrowAnonymousFunction-test/unit_tests/callbackchainforloop.js-8-16","t_140":"arrowAnonymousFunction-test/unit_tests/callbackchainforloop.js-13-14"}}